spec/support/factory_helper.rb

```diff
@@ -1,12 +1,11 @@
 # I'm currently using Fabrication gem to generate objects but if this changes now I can make the change in one place
```

Comment:
I wasn't totally sure whether I should do it either but it may save me some effort in the future and it helps to reduce coupling which is something I'm trying to be mindful of when writing my code.